docs/_partials/billing/enable-billing.mdx

```diff
@@ -8,5 +8,5 @@ Clerk billing costs just 0.7% per transaction, plus Stripe's transaction fees wh
 
 Once you have enabled billing, you will see the following **Payment gateway** options for collecting payments via Stripe:
 
-- **Clerk development gateway**: A shared **test** Stripe account so developers can get started testing and building with billing **in development** without needing to create and configure a Stripe account.
-- **Stripe account**: Use your own Stripe account.
+- **Clerk development gateway**: A shared **test** Stripe account used for development instances. This allows developers to test and build billing flows **in development** without needing to create and configure a Stripe account.
+- **Stripe account**: Use your own Stripe account for production. **A Stripe account created for a development instance cannot be used for production**. You will need to create a separate Stripe account for your production environment.
```

Comment:
>Use your own Stripe account for production.

This implies a bit to me that you can't create your own Stripe account for development instances, but I guess the following sentences clarify 🤔

Reply:
>But, are you saying that one could create a Stripe account for development, and not use the one created by Clerk?

Yep, but I think the text here works.